<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CChooseDialog</class>
 <widget class="QDialog" name="CChooseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CChooseDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>模型选择</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>应用</string>
    </property>
    <widget class="QComboBox" name="appComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>21</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>组</string>
    </property>
    <widget class="QComboBox" name="groupComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>型号</string>
    </property>
    <widget class="QComboBox" name="modelComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="surePushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelPushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>CancelPushButton</zorder>
  <zorder>surePushButton</zorder>
  <zorder>CancelPushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>surePushButton</sender>
   <signal>clicked()</signal>
   <receiver>CChooseDialog</receiver>
   <slot>slot_clickSure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_clickSure()</slot>
 </slots>
</ui>
